commands/backups/capture.js

```diff
@@ -12,6 +12,22 @@ function * run (context, heroku) {
   const interval = Math.max(3, parseInt(flags['wait-interval'])) || 3
   const db = yield fetcher.addon(app, args.database)
 
+  let dbInfo = yield heroku.request({
+    host: host(db),
+    method: 'get',
+    path: `/client/v11/databases/${db.id}`
+  }).catch(err => {
+    if (err.statusCode !== 404) throw err
+    cli.exit(1, `${cli.color.addon(db.name)} is not yet provisioned.\nRun ${cli.color.cmd('heroku addons:wait')} to wait until the db is provisioned.`)
+  })
+
+  if (dbInfo) {
+    let dbProtected = /On/.test(dbInfo.info.find(attribute => attribute.name === 'Continuous Protection').values[0])
+    if (dbProtected) {
+      yield cli.confirmApp(app, flags.confirm, 'Continuous protection is already enabled for this database. Logical backups of large databases are likely to timeout. See https://devcenter.heroku.com/articles/heroku-postgres-data-safety-and-continuous-protection#physical-backups-on-heroku-postgres.  Are you sure?')
```

Comment:
instead of "likely to timeout", can we mention something like it can result to fail... uhmm due to errors? or "likely to fail"?
we're seeing two issues with the large database: 1) timing out because taking a logical backup can't finish within 24 hours 2) some OOM on either dyno side or postgres side or something that just impossible to take a backup. While both of them are the failure, actually for the case of 1), you don't really see "timeout" as the result (to the user, it either marked as "failed" without obvious reason or the state doesn't change from "running"), so I thought that mentioning "timeout" can be confusing.